thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -37,6 +39,22 @@ export default Component.extend({
    */
   optionsMapping: ANOMALY_OPTIONS_MAPPING,
 
+  /**
+   * Urn of an anomaly
+   * @type {String}
+   */
+  anomalyUrn: computed(
+    'anomalyUrns',
+    function () {
+      const { anomalyUrns } = getProperties(this, 'anomalyUrns');
```

Comment:
Are you expecting more values in the future? Seems pointless to use `getProperties` on a single item and then have to parse the resulting object.

Reply:
Nope, I'll make the fix.